packages/framework/resources/views/layouts/styles.blade.php

```diff
@@ -2,17 +2,22 @@
 <style>[x-cloak] {display: none!important}</style>
 
 {{-- The compiled Tailwind/App styles --}}
-@if(config('hyde.load_app_styles_from_cdn', false))
-    <link rel="stylesheet" href="{{ HydeFront::cdnLink('app.css') }}">
-@elseif(Asset::exists('app.css'))
-    <link rel="stylesheet" href="{{ Asset::get('app.css') }}">
-@endif
+@if(Vite::running())
+    {{ Vite::assets(['resources/assets/app.css']) }}
+@else
+    @if(config('hyde.load_app_styles_from_cdn', false))
+        <link rel="stylesheet" href="{{ HydeFront::cdnLink('app.css') }}">
+    @elseif(Asset::exists('app.css'))
+        <link rel="stylesheet" href="{{ Asset::get('app.css') }}">
+    @endif
+
```

Comment:
Would it make more sense if we just automatically enabled Vite when using `Asset::get` and Vite is running?